### DISCUSSION TITLE
URGENT HELP - locked out of account - 5 days awaiting for a reply

### DISCUSSION THREAD

**Discussion by GabriellyCastro-Moni** [community/community#37576] [Other Feature Feedback, Questions, & Ideas]
Hi team, I'm locked out of my work account and I need to get back in ASAP. I have lost my recovery codes and need to reset my authentication. 

I've raised a ticket 4 days ago and I'm still awaiting a reply. I need to get back to work ASAP. Please help? 
Ticket ID: 1846185

**Comment by GabriellyCastro-Moni** [comment]
Ticket ID: 1846185  - raised 4 days ago. I lost my recovery codes and need to get back in...

**Comment by Wep-wep** [comment]
Prueba con otra cosa

**Comment by queenofcorgis** [comment]
Hi there! I checked in on your ticket, it looks like it has been solved. Please be advised the Community team is unable to assist with Support wait times nor escalate tickets.

**Comment by rohan717** [comment]
Hello here,
Is there any solution for this issue? I'm also facing the same problem.
I lost my recovery codes and 2FA device. Now I can't login to my gitHub account. this is for my work email and it's absolutely crucial.
please help me with this
Ticket ID: 1908643 /1908551 
Thank you..!!

**Reply by queenofcorgis** [reply]
Hi there! As stated above - [Support](https://support.github.com/) is the only place that can assist, the Community is unable to help nor escalate. Going to lock this post now as the only avenue for being locked out of your account is through Support. Please be patient as our team has a process for reviewing tickets.
